Dockerfile-linux.template

```diff
@@ -238,6 +238,8 @@ RUN set -eux; \
 
 COPY docker-php-source /usr/local/bin/
 
+ARG PHP_BUILD_PROVIDER="Docker Official Images"
+
```

Comment:
Would it be possible for the actual builders to set this argument? This would allow the actually official builds to carry the tag, whereas and manual builds of the Dockerfiles do not.